TGC/events/Filibuster.txt

```diff
@@ -0,0 +1,623 @@
+						add_core = FIL
+						secede_province = FIL
+					}
+					#2197 = {add_core = FIL}
+					2199 = {add_core = FIL}
+					#capital = 2199	
+				}
+
+				FIL = {
+					activate_technology = military_staff_system
+					activate_technology = iron_muzzle_loaded_artillery
+
+				}
+
+
+				188 = {
+					soldiers = {
+						#limit = {
+						#	has_pop_culture = dixie
+						#}
+						move_pop = 2198
+					}
+					officers = {
+						#limit = {
+						#	has_pop_culture = dixie
+						#}
+						move_pop = 2198
+					}
+				}
+
+				FIL = {
+					define_general = {
+						name = "William Walker"
+						personality = adventurer
+						background = relentless
+					}
+					money = 10000
+					[firearms] = 69
+				}
+
+				war = {
+					target = FIL
+					attacker_goal = { casus_belli = acquire_all_cores country = NIC }
+					defender_goal = { casus_belli = acquire_all_cores country = FIL }
+					call_ally = no
+				}
+
+				ai_chance = { factor = 1 }
+			}
+
+			option = {
+				name = "EVTOPTB8870780"
+
+				prestige = -5
+				NIC = {
+					any_owned = {
+						limit = { province_id = 2198 }
+						add_core = FIL
+						secede_province = FIL
+					}
+					any_owned = {
+						limit = { province_id = 2197 }
+						add_core = FIL
+						secede_province = FIL
+					}
+					#2197 = {add_core = FIL}
+					2199 = {add_core = FIL}
+					#capital = 2199	
+				}
+
+				FIL = {
+					activate_technology = military_staff_system
+					activate_technology = iron_muzzle_loaded_artillery
+				}
+
+
+				188 = {
+					soldiers = {
+						#limit = {
+						#	has_pop_culture = dixie
+						#}
+						move_pop = 2198
+					}
+					officers = {
+						#limit = {
+						#	has_pop_culture = dixie
+						#}
+						move_pop = 2198
+					}
+				}
+
+				FIL = {
+					define_general = {
+						name = "William Walker"
+						personality = adventurer
+						background = relentless
+					}
+					money = 10000
+					[firearms] = 69
+				}
+
+				war = {
+					target = FIL
+					attacker_goal = { casus_belli = acquire_all_cores country = NIC }
+					defender_goal = { casus_belli = acquire_all_cores country = FIL }
+					call_ally = no
+				}
+				change_tag_no_core_switch = FIL
+				ai_chance = { factor = 0 }
+			}
+		}
+
+	## USA gets choice to recognize Filibuster state
+		country_event = {
+			id = 8870781
+			title = "EVTNAME8870781"
+			desc = "EVTDESC8870781"
+			picture = "Expansion"
+
+			is_triggered_only = yes
+
+			option = {
+				name = "EVTOPTA8870781"
+				badboy = -1
+				prestige = -5
+				FIL = {
+					country_event = {
+						id = 8870782
+					}
+				}
+				ai_chance = { factor = 0.15 }
+			}
+
+			option = {
+				name = "EVTOPTB8870781"
+				prestige = 5
+				ai_chance = { factor = 0.7 }
+				FIL = {
+					country_event = {
+						id = 8870783
+					}
+				}
+			}
+
+			option = {
+				name = "EVTOPTC8870781"
+
+				badboy = 2
+				create_alliance = FIL
+
+				diplomatic_influence = {
+					who = FIL
+					value = 400
+				}
+
+				ai_chance = { factor = 0.15 }
```

Comment:
Nitpick: try to always have the ai_chance tag at the bottom of an option for easier readability.